init.ts

```diff
@@ -86,6 +88,12 @@ app.enable('trust proxy');
 
 const doRunTests = (process.env.RUN_TESTS || '').trim() === '1';
 
+// we have to load some mocks before the app starts...
+if (doRunTests) {
+	console.log('Loading mocks...');
+	import('./test/test-lib/aws-mock');
```

Comment:
Can this whole block be moved into https://github.com/balena-io/open-balena-api/blob/master/test/00-init.ts?

I'm now seeing you already do the import there -- can you please remove this one?

Reply:
@dfunckt I tried this, but it does not work if I only import it there. I stole this from the PR from rich here: https://github.com/balena-io/open-balena-api/pull/143/files

Reply:
Can we instead create a new file in `tests/`, put all pre-run code there and require it from here? The end result is practically the same, but will not encourage our future selves to add more and more test related code, which just doesn't fit here.

Reply:
That sound good! Will refactor it like this.